assets/js/vue-apps/grant-filters.js

```diff
@@ -0,0 +1,93 @@
+import $ from 'jquery';
+import Vue from 'vue';
+
+function init() {
+    const mountEl = document.getElementById('js-grant-filters');
+    if (!mountEl) {
+        return;
+    }
+    new Vue({
+        el: mountEl,
+        delimiters: ['<%', '%>'],
+        data() {
+            let facets = {};
+            let defaultFilters = {
+                amount: '',
+                year: '',
+                programme: '',
+                country: '',
+                localAuthority: ''
+            };
+            let filters = Object.assign({}, defaultFilters);
+            return {
+                facets,
+                defaultFilters,
+                filters
+            };
+        },
+        mounted: function() {
+            // Enable inputs (they're disabled by default to avoid double inputs for non-JS users)
+            $(this.$el)
+                .find('[disabled]')
+                .removeAttr('disabled');
+            // Populate the facets object
+            let facets = this.$el.getAttribute('data-facets');
+            if (facets) {
```

Comment:
I'm a little wary of passing JSON objects through as data params and re-parsing. I wonder if exposing this data on the page as global window variables might work? It's still a hack, but maybe avoids parsing a big object through an attribute.

Reply:
Yeah, fair point. Have moved it to a global object which saves re-parsing JSON.